obspy/mseed/src/libmseed/msrutils.c

```diff
@@ -358,7 +358,16 @@ msr_normalize_header ( MSRecord *msr, flag verbose )
 	  sec = msr->starttime / (HPTMODULUS / 10000);
 	  usec = msr->starttime - (sec * (HPTMODULUS / 10000));
 	  usec /= (HPTMODULUS / 1000000);
-	  
+
+      /* Deal with negative timestamps and a negative offset.
```

Comment:
The indent is wrong here (though you probably didn't notice because libmseed uses strange mixed tab/space indents.)